setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup, find_packages
+
+setup(name='home-assistant-frontend',
+      version='20171021.0',
+      description='The Home Assistant frontend',
+      url='https://github.com/home-assistant/home-assistant-polymer',
+      author='Paulus Schoutsen',
+      author_email='Paulus@PaulusSchoutsen.nl',
+      license='Apache License 2.0',
+      packages=find_packages(include=['hass_frontend', 'hass_frontend.*']),
+      zip_safe=False)
```

Comment:
Why isn't it zip safe?

Reply:
> For maximum performance, Python packages are best installed as zip files. Not all packages, however, are capable of running in compressed form, because they may expect to be able to access either source code or data files as normal operating system files. So, setuptools can install your project as a zipfile or a directory, and its default choice is determined by the project's zip_safe flag.

http://peak.telecommunity.com/DevCenter/setuptools#setting-the-zip-safe-flag

We serve frontend files from this package.